src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs

```diff
@@ -107,6 +108,15 @@ private void LoginButton_OnClick(object sender, RoutedEventArgs e)
                 }
             }
         }
+
+        public bool EnableItems
```

Comment:
Please follow https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members?redirectedfrom=MSDN#names-of-properties, I would suggest `AreMenuItemsEnabled` here

Reply:
Done.